### ISSUE TITLE
Using % placed in changed loc_limit_QS example does not produce net losses as expected

### ISSUE THREAD

**Issue by aiste-kalinauskaite** [OasisLMF/ReinsuranceTestTool#33]
I've changed loc_limit_QS example as follows:

% ceded = 1
% placed = 0.8

I would have expected losses ceded to reinsurer to be 10 x 4 x 0.8 (risk limit * number of locations * % placed) = 32. I got 25.6 (net loss = 374.4).

**Comment by johcarter** [comment]
The reason is the percent placed is being applied at the risk level and the portfolio level. it should only be applied at the final calculation level.
In general, where there are two levels of calculation, then it should be applied at the 2nd level. Where there is 1 level of calculation (not including the dummy calculation to aggregate loss to portfolio level and pass through losses), it should be applied at the first level.

**Comment by johcarter** [comment]
Quota share will always have two calculations as the risk level will not be SEL.  The percent placed always applies at the 2nd calculation only, with the occurrence limit.
Please see new examples placed_acc_QS and placed_acc_1_QS

**Comment by johcarter** [comment]
Surplus share. Currently the calculation is done in 1 step with a passthrough rule at level 2. Percent placed is being applied at level 1.
However if there is an occurrence limit then an additional rule is being applied at level 2 (calcrule 23). in this situation the percent placed must be applied in calcrule 23 at level 2 and not at level 1.

**Comment by johcarter** [comment]
Surplus share with percent placed and occ limit: please see placed_loc_SS and placed_loc_limit_SS

**Comment by johcarter** [comment]
This is now fixed in v0.4 (dev). @aiste-kalinauskaite please can you retest and close issue if happy? thanks

**Comment by aiste-kalinauskaite** [comment]
My earlier provided example now works as expected
